base/applications/osk/keyboard.c

```diff
@@ -0,0 +1,251 @@
+/*
+ * PROJECT:         ReactOS On-Screen Keyboard
+ * LICENSE:         GPL - See COPYING in the top level directory
```

Comment:
https://reactos.org/wiki/Coding_Style#File_Structure

Reply:
This was copied from `main.c` to keep consistency